.eslintrc.js

```diff
@@ -0,0 +1,45 @@
+module.exports = {
+	extends: ['eslint:recommended', 'prettier', 'prettier/react'],
+	plugins: ['react', 'prettier'],
+	parser: 'babel-eslint',
+	parserOptions: {
+		ecmaVersion: 2016,
+		sourceType: 'module',
+		ecmaFeatures: {
+			jsx: true
+		}
+	},
+	env: {
+		browser: true,
+		node: true,
+		es6: true
+	},
+	rules: {
+		curly: ['error', 'all'],
+		'prettier/prettier': [
+			'error',
+			{
+				printWidth: 120,
+				semi: false,
+				singleQuote: true,
+				trailingComma: 'none', // ideally 'all'...
+				tabWidth: 1,
+				useTabs: true
```

Comment:
I noticed you mainly used tabs, but if you want to use 2 spaces (hint hint), set:

```js
{
  tabWidth: 2,
  useTabs: false
}
```